build-parent/pom.xml

```diff
@@ -27,6 +27,7 @@
         <scala.version>2.12.8</scala.version>
         <scala-plugin.version>4.1.1</scala-plugin.version>
 
+        <version.enforcer.plugin>3.0.0-M3</version.enforcer.plugin>
```

Comment:
Is this used anywhere?

Reply:
It is overriding the version from `jboss-parent`, just like the surefire property below.

Reply:
Ah! Thanks!